content/04.use-cases-b.md

```diff
@@ -9,6 +9,7 @@ The "Issues" feature of GitHub allows for discrete tasks and sub-tasks to be ide
 Scripts, commit messages, and pull requests can be linked directly to issues and discussions, providing a clear record of project workflow.
 The use of GitHub for all project-related conversation and planning, rather than e-mail or messaging tools, makes it easier to keep track of progress throughout the lifespan of a project and less likely for issues to slip through the cracks.
 It is not essential for all team members to have proficiency in git or programming, as users can interact with Issues and Discussions via web browser or e-mail.
+GitHub can also be integrated with other project management software like Slack or Zenhub.
```

Comment:
Great! Yes,  I can definitely add some writing re: using github integrations for project management.